Makefile

```diff
@@ -87,29 +84,38 @@ version:
 	@echo $(VERSION)
 
 ./bin/log-counter: $(PKG_SOURCES)
+ifeq ($(ENABLE_JOURNALD), 1)
 	CGO_ENABLED=$(CGO_ENABLED) GOOS=linux GO111MODULE=on go build \
 		-mod vendor \
 		-o bin/log-counter \
 		-ldflags '-X $(PKG)/pkg/version.version=$(VERSION)' \
-		$(BUILD_TAGS) \
+		-tags "$(BUILD_TAGS)" \
 		cmd/logcounter/log_counter.go
+else
+	echo "log-counter requires journald, skipping."
```

Comment:
Nit: Print this as a warning:

echo "Warning: ..."

Reply:
Good point. Thanks!